UIViewPreviewProvider.swift

```diff
@@ -100,6 +104,7 @@ extension UIViewPreviewProvider {
         
         return ForEach(identifiablePreviews) { preview in
             swiftUIView(from: preview.item)
+                .environment(\.colorScheme, colorScheme)
         }
     }
```

Comment:
Then down here we can iterate over all the color schemes and make a preview for each of them
```swift
    public static var previews: some View {
        
        let identifiablePreviews = uiPreviews.map {
            IdentifiableBox($0)
        }
        
        let identifieableSchemes = colorSchemes.map {
            IdentifiableBox($0)
        }
        
        return ForEach(identifiablePreviews) { preview in
            ForEach(identifieableSchemes) { scheme in
                swiftUIView(from: preview.item)
                    .environment(\.colorScheme, scheme.item)
            }
        }
    }
```